omod/src/main/java/org/openmrs/web/controller/PortletController.java

```diff
@@ -399,30 +479,56 @@ public ModelAndView handleRequest(HttpServletRequest request, HttpServletRespons
 						Concept c = cs.getConcept(i);
 						concepts.put(i, c);
 						conceptsByStringIds.put(i.toString(), c);
-					}
-					catch (Exception ex) {
+					} catch (Exception ex) {
 						log.error("Error during putting int i into concept c", ex);
 					}
 				}
-				
+
 				model.put("conceptMap", concepts);
 				model.put("conceptMapByStringIds", conceptsByStringIds);
 			}
-			
+
 			populateModel(request, model);
 			log.debug(portletPath + " took " + (System.currentTimeMillis() - timeAtStart) + " ms");
 		}
-		
+
 		return new ModelAndView(portletPath, "model", model);
-		
+
 	}
-	
+
 	/**
-	 * Subclasses should override this to put more data into the model. This will be called AFTER
-	 * handleRequest has put mappings in the model as described in its javadoc. Note that context
+	 * Subclasses should override this to put more data into the model. This will be
+	 * called AFTER
+	 * handleRequest has put mappings in the model as described in its javadoc. Note
+	 * that context
 	 * could be null when this method is called.
 	 */
 	protected void populateModel(HttpServletRequest request, Map<String, Object> model) {
 	}
-	
+
+	private Integer getPageParameter(HttpServletRequest request) {
+		try {
+			return Integer.parseInt(request.getParameter("page"));
```

Comment:
Can you take a look at our naming convention for the pagination parameters? https://openmrs.atlassian.net/wiki/spaces/docs/pages/25469830/REST+Web+Services+API+For+Clients#Limiting-the-number-of-results-and-paging

Reply:
Basing on the user interface screen that this controller serves, could you remind me on how this parameter is changed to a different value?

Reply:
So the method retrieves the page parameter from the HTTP request, and if it'ss missing or is invalid like non numeric it will default to 0. The page value calculates which subset of data (encounters or observations) to fetch `patientDashboard?page=2`

Reply:
What i mean is, how is this parameter changed in the HTTP request by the user interface?

Reply:
With the dropdown selector

Reply:
Does it select page or pageSize?

Reply:
Can you share a screenshot of that entire page?

Reply:
![image](https://github.com/user-attachments/assets/e064431b-8d4c-4661-97ef-7854806a3a55)

Reply:
Awesome!!! 👍

Reply:
Did you check the link about naming conventions?

Reply:
Yes, I have changed that, please review